src/main/java/com/fortanix/sdkms/performance/sampler/KeyGenerationSampler.java

```diff
@@ -79,9 +82,21 @@ public Object execute() throws ApiException {
 
     @Override
     public void teardownTest(JavaSamplerContext context) {
+        this.login(); /* Forcefully re-authenticates thread in case the idle-timeout has kicked
```

Comment:
Will SDKMS Auth API returns same bearer token on multiple calls? If **not** then this is not thread safe, and hence the bearer token may be invalid due to login by another thread. Hence make sure one of the either:
1. login method is threadsafe
2. add some retries to deleteSecurityObject call in case of expiry failure.